config/locales/en.yml

```diff
@@ -910,6 +918,10 @@ en:
           attributes:
             plan:
               not_allowed: "not allowed in this context"
+            keys:
+              invalid: Invalid. Use only alphanumeric characters [0-9, a-z, A-Z], hyphen-minus (-), no spaces and 40 characters at most.
```

Comment:
Where is `40` coming from?

The validation is for [5..255 characters](https://github.com/3scale/porta/blob/b122bced37550ea23661cee1e7a13c4ccd50d105/app/models/application_key.rb#L15) and it used to say "256 characters at most".

Also, `/\A[\x20-\x7E]+\Z/` includes a bunch of "special" characters, so we need to clarify that als. And in fact - we need to restrict the validation rules because some characters should not be allowed - but there is another JIRA for this, I think.

Reply:
The other JIRA that I was thinking of seems to be this one: https://issues.redhat.com/browse/THREESCALE-10002

Reply:
It was like this already, who knows. I updated it.

Reply:
The validation is bogus, the following key is actually valid `%%%%%` 🤷🏻 but nobody seems to care